pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/OncHistoryDetail.java

```diff
@@ -396,7 +399,7 @@ public static OncHistoryDetail getOncHistoryDetail(@NonNull String oncHistoryDet
         return (OncHistoryDetail) results.resultValue;
     }
 
-    public static String createNewOncHistoryDetail(@NonNull String medicalRecordId, @NonNull String changedBy) {
+    public static String createNewOncHistoryDetail(@NonNull String medicalRecordId, @NonNull String changedBy, Patch patch) {
```

Comment:
this method is called only by `OncHistoryDetailPatch`, since patch already does upsert to ES, do we need to add upsert complexity to this method as well?